src/sqlfluff/rules/L014.py

```diff
@@ -37,9 +37,10 @@ class Rule_L014(Rule_L010):
 
     The functionality for this rule is inherited from :obj:`Rule_L010`.
```

Comment:
I wonder if we should remove this (and all other references to rules inheriting from one another).

Reply:
Agreed. All 3 are gone.